### DISCUSSION TITLE
How does Truvari deal with translocations?

### DISCUSSION THREAD

**Discussion by Thomas-Rauter** [spiralgenetics/truvari#169] [Q&A]
Hello,

I am studying genomics and am using Truvari for benchmarking SV caller results. I'm curious about how Truvari benchmarks translocations, specifically regarding the refdist parameter (refdist | 500 | Maximum distance comparison calls must be within from base call's start/end). Given that translocations have two start/end position pairs, does Truvari apply the refdist parameter to both pairs?

I've examined the source code and tried benchmarking SV callers, but I couldn't clarify this aspect. Any insights or recommendations would be much appreciated.

Thank you,

Thomas

**Comment by ACEnglish** [comment]
Hello,

I'm sorry to say that Truvari does not handle translocation (or any BNDs). I also don't have them included on my development roadmap. 

Have a good day,
~/Adam

**Reply by Thomas-Rauter** [reply]
Thank you very much for your quick answer!

Have a good day too,
Thomas
